zipline/pipeline/visualize.py

```diff
@@ -174,10 +174,7 @@ def writeln(f, s):
 
 def fmt(obj):
     if isinstance(obj, Term):
-        if hasattr(obj, 'short_repr'):
-            r = obj.short_repr()
-        else:
-            r = type(obj).__name__
+        r = obj.graph_repr()
     else:
         r = obj
     return '"%s"' % r
```

Comment:
This formatting is broken for strings. If you want to repr strings with quotes use `%r` I wouldn't block on this comment though.